### PR TITLE
Announcement: Dasharo User Group #1

### PR DESCRIPTION
**Pull request by andrewdavidwong**
How does this look, @pietrushnic?

CC: @marmarek

### CODE REVIEW

File: 2023-03-15-dasharo-user-group-1.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+**The Dasharo Developers vPub** is a virtual event with goal for gathering developers and enthusiasts in the technology industry to engage in discussions and conversations related to open-source firmware and other related topics. The vPub is designed to be a less structured, more relaxed environment where participants can share knowledge and ideas, connect with others in the industry, and learn about the latest developments in the field. The conversations can range from technical challenges faced by developers, off-topic discussions, to new features or improvements. These events have brought together experts and enthusiasts from all over the world, providing valuable opportunities for learning, collaboration, and networking. Everything in casual atmosphere with beverage of your choice in your hand.
+
+[Read the full announcement for more information.](https://vpub.dasharo.com/e/1/dasharo-user-group-1)
```
Comment (pietrushnic) [inline 2023-03-15-dasharo-user-group-1.md:21]: @andrewdavidwong does it make sense to mention MSI certifcation or internally link to it?

**Comment by marmarek** [comment]
It lacks a context why this event might be relevant for Qubes community. As @pietrushnic mentioned, it would be good to add relation to MSI certification, but maybe also planned interview in the vPub part?

**Comment by andrewdavidwong** [comment]
> It lacks a context why this event might be relevant for Qubes community. As @pietrushnic mentioned, it would be good to add relation to MSI certification, but maybe also planned interview in the vPub part?

Good point. Just rewrote the announcement. How does it look now?

**Review by marmarek — APPROVED** [review]
```
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA256

Approved as of ed30a51fa530166d445eb74df06138de1a13ad4d
-----BEGIN PGP SIGNATURE-----

iQIzBAEBCAAdFiEEnYWYU8Ll1fIX9iss8yiUvpaEk4oFAmQR79sACgkQ8yiUvpaE
k4o+zxAAhTjOpTVclTZ27PHoPibMF3d6F/5EfkOoQKAEzMFo8WEqURGtLM8OLGbx
DyYzPK3DNw0//0YrCD5L954+OGIDeWEEOA2F4pKC4kHW87fjgJqiwVV2j9Jq0RHy
Y5ZVcCuroMdaPUHXcErNq0VjJ974jAzGROcp4qVHtILYyHoPhfS6lYcGyC39W+Xm
IpMySyG0a+Y3bhA3zp5V1mLa8M5/eXldyDaTb9PPFqZO6IXPUAHIjfrtmuP/NWcw
wKsVekllxkKiRVfk0P2td+zG9vE26TvLPh/rpjeTmjhApSj87zblBDmU7i4mDxfL
yk6UkYCazFVfh9lLfPZovA9nO/qMY4nlebPEaw2lHbiWmdczvF/Rn1aw2k1HPwLG
7S/X8DnsrovZlJl/f+gQyjbeBxUzgAjvE24m84tT7L+97qbsylFY4NaFWKYGTHOR
jYQ6ML385+Ojao4p81ExV4p6LOHGW584/BGUK+BIOf1xkRVp1hCsDLvEZfB8A3x7
C0UsFgrKIUTDKXKNusbcEpJAEQeAUK1XLd8L62peEInYkAvLc7aZb3xEevyb3aIS
OrLgkpvGVqKFMlwfz4h23zIJQoNIcnfI6IkEBKzCDBgenkvCc5JfV/ENoJVzR3ul
xqKVhem2s3hDln+YmQMJ5IksR7sir+7CQRjVWtNhDvW4yMPQxrc=
=UXwN
-----END PGP SIGNATURE-----
```

**Pull request merged**